flan-t5-xl/flan-t5-xl-trt-llm/config.yaml

```diff
@@ -0,0 +1,21 @@
+base_image:
+  image: bolabaseten/tensorrt-llm-python:latest
+  python_executable_path: /usr/bin/python3
+environment_variables: {}
+external_package_dirs: []
+model_metadata:
+  repo_id: htrivedi99/flan-t5-xl-trt-test
+model_name: Flan T5 XL TRT
+requirements:
+  - mpi4py==3.1.5
+  - pynvml==11.5.0
+resources:
+  accelerator: A100
+  use_gpu: true
+secrets:
+  hf_access_token: HF_TOKEN_HERE
+runtime:
```

Comment:
This really high number is usually good for continuous batching. This doesn't seem to support that right now. @joostinyi @aspctu what would be a good number here?